app/styles/components/universal-selection.less

```diff
@@ -0,0 +1,54 @@
+.universal-selection {
+  .power-select-option {
+    display: flex; 
+    align-items: center;
+  }
+
+  .power-select-option__name {
+    font-weight: 700;
+    padding-left: @spacing-xxx-sm;
+    flex: 3;
+  }
+
+  .power-select-option__influencers-count {
+    text-align: right;
+    color: @upf-feedback-info;
+    flex: 1;
+  }
+
+  .power-select-option__icon {
+    color: @upf-primary-bright-purple;
+  }
+
+  .ember-power-select-option {
```

Comment:
all these `important` are actually due to the fact that you are using a dependency's inner class directly, which is fixed by having a more accurate class naming. (and generally, having that many `important` should be seen as a red flag)

Reply:
any ideas how to override without the important? i dislike the important too but couldnt figure out a way to be more specified and override those styling

Reply:
oh actually if i rename `universal-selection-option` , then that might work for this issue ?